install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2947,6 +2947,12 @@ sctp:
 
 # Configuration for types of authentication for Cilium
 authentication:
+  # -- Buffer size of the channel Cilium uses to receive authentication events from the signal map.
+  queueSize: 1024
+  # -- Buffer size of the channel Cilium uses to receive certificate expiration events from auth handlers.
+  rotatedIdentitiesQueueSize: 1024
+    # -- Interval for garbage collect expired auth map entries.
```

Nit:

```suggestion
    # -- Interval for garbage collection of expired auth map entries.
```

or “for garbage-collecting”

---

@qmonnet changed - thanks!